src/databricks/labs/ucx/cli.py

```diff
@@ -652,7 +653,7 @@ def assign_owner_group(
     else:
```

Comment:
We like to minimize the logic in the cli module, I will accept these changes, be mindful about this